### PR TITLE
SDI-18: 🔧 Remove old (unused) ingress

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: charts/generic-service/templates/ingress_v1_2.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.ingress.enabled .Values.ingress.v1_2_enabled -}}
```
Comment (petergphillips) [inline charts/generic-service/templates/ingress_v1_2.yaml:1]: everyone will have moved off by now as old ingress doesn't exist anymore

File: charts/generic-service/values.yaml [hunk 2]
```diff
@@ -43,11 +43,6 @@ service:
 
 ingress:
   enabled: false
-  # v1.2 is the new ingress controller, replacing the existing 0.47 ones
-  # instructions on advised upgrade process - https://mojdt.slack.com/archives/CH6D099DF/p1657794137979789
-  # switching this to true with perform a big bang ingress switch with possible downtime as dns changes
-  v1_2_enabled: false
-  v0_47_enabled: true
```
Comment (neilmendum) [inline charts/generic-service/values.yaml:50]: Is removing these a breaking change for services that use the `generic-service`? They would have something like this in their `values.yaml`
```yaml
generic-service:
  ingress:
    enabled: true
    v1_2_enabled: true
    v0_47_enabled: false
```
If so wondering if we want to bump the major version to `3.0.0`?

Comment (petergphillips) [reply charts/generic-service/values.yaml:50]: no, should have no impact on existing clients.  Values will just be ignored.

Comment (neilmendum) [reply charts/generic-service/values.yaml:50]: As discussed offline - with https://github.com/ministryofjustice/hmpps-helm-charts/pull/69 we removed `contextColour` while numerous projects are still defining it and they didn't break, so seems like Helm is not strict about variables and removing these isn't a breaking change

**Review by neilmendum — APPROVED** [review]
LGTM

**Pull request merged**